community/portworx/values.yaml

```diff
@@ -24,7 +27,7 @@ storkVersion: 2.9.0
 
 customRegistryURL: 
 registrySecret:
-imagePullSecrets:
```

Comment:
Why we don't need `imagePullSecrets`?

Reply:
it's a weird parameter in this chart, registry secret is used by px and stork, imagepullSecrets is used by other images. in operator we only support same secret for all images.